pkg/sentry/mm/vma.go

```diff
@@ -195,7 +195,9 @@ func (mm *MemoryManager) applicationAddrRange() usermem.AddrRange {
 
 // Preconditions: mm.mappingMu must be locked.
 func (mm *MemoryManager) findLowestAvailableLocked(length, alignment uint64, bounds usermem.AddrRange) (usermem.Addr, error) {
-	for gap := mm.vmas.LowerBoundGap(bounds.Start); gap.Ok() && gap.Start() < bounds.End; gap = gap.NextGap() {
+	// use new function to speed up the search
+	//for gap := mm.vmas.LowerBoundGap(bounds.Start); gap.Ok() && gap.Start() < bounds.End; gap = gap.NextGap() {
```

Comment:
No need to leave dead code like this.

It's a bit odd that the first gap checked might not be large enough, but I don't personally object to this API.

Reply:
I revert it. It was a wierd result but now it's correct.